### PR TITLE
feat: Added 3 dot menu to delete helm app

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes [AB#1229](https://dev.azure.com/DevtronLabs/Devtron/_boards/board/t/Devtron%20Team/Issues/?workitem=1229)

Screenshot 2022-12-21 at 2 09 49 PM

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Testing is done manually 

- [x] Create and delete helm app through UI
- [x] Create an external helm app & delete

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx [hunk 1]
```diff
@@ -68,6 +76,47 @@ function EnvironmentSelectorComponent() {
...
+                    }}
+                >
+                    <span>Delete</span>
+                    <Trash className="icon-dim-20 scr-5" />
+                </span>
+            </div>
+        )
+    }
+
+    const getDeleteApplicationApi = (force?: boolean): Promise<any> => {
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx:95]: We can remove force param if unused

File: src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx [hunk 2]
```diff
@@ -68,6 +76,47 @@ function EnvironmentSelectorComponent() {
         showUrlInfo(true)
     }
 
+    const Popup = () => {
+        return (
+            <div className="pod-info__popup-container">
+                <span
+                    className="flex pod-info__popup-row pod-info__popup-row--red cr-5"
+                    onClick={(e) => {
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx:84]: Extract this out to separate function

File: src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx (hunk 1 above)
Comment (milindk8) [reply src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx]: done

File: src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx (hunk 2 above)
Comment (milindk8) [reply src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx]: done

File: src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx [hunk 3]
```diff
@@ -16,8 +16,15 @@ import ScaleWorkloadsModal from './scaleWorkloads/ScaleWorkloadsModal.component'
...
+import { ReactComponent as Trash } from '../../../../assets/icons/ic-delete-interactive.svg'
+import { deleteApplicationRelease } from '../../../external-apps/ExternalAppService';
+import { deleteInstalledChart } from '../../../charts/charts.service';
+import { toast } from 'react-toastify';
+import { ReactComponent as Dots} from '../../assets/icons/ic-menu-dot.svg'
+import { DeleteChartDialog } from '../../values/chartValuesDiff/ChartValuesView.component';
+import { checkIfDevtronOperatorHelmRelease, URLS } from '../../../../config';
 
-function EnvironmentSelectorComponent() {
+function EnvironmentSelectorComponent({isExternalApp}: {isExternalApp: boolean}) {
```
Comment (shivani170) [inline src/components/v2/appDetails/sourceInfo/EnvironmentSelector.component.tsx:27]: type of isExternalApp is missing

**Review by shivani170 — APPROVED** [review]

**Pull request merged**